app/views/sessions/new.mobile.haml

```diff
@@ -2,42 +2,37 @@
 -#   licensed under the Affero General Public License version 3 or later.  See
 -#   the COPYRIGHT file.
 
-.session_mobile
-  .landing
-    %h1.session
-      = pod_name
-
-  #main_stream.stream
-    - flash.each do |name, msg|
-      .expose#flash-container
-        .flash-message{class: "message alert alert-#{flash_class name}"}
-          = msg
-
-    #login_form
-      .login-container
-        = form_for(resource, as: resource_name, url: session_path(resource_name)) do |f|
-          %fieldset
-            %legend
-              = image_tag("branding/logos/header-logo2x.png", height: 40, width: 40)
-              = t("devise.sessions.new.login")
-
-            .form-group
-              = f.label :username, t("username")
-              = f.text_field :username, autofocus: true, class: "form-control", autocapitalize: "none", autocorrect: "off"
-
-            .form-group
-              = f.label :password , t("password")
-              = f.password_field :password, class: "form-control"
-              = hidden_field(:user, :remember_me, value: 1)
-
-            .form-group
-              = f.submit t("devise.sessions.new.sign_in"), class: "btn btn-primary"
-              - if display_registration_link?
-                = link_to t("devise.shared.links.sign_up"), new_registration_path(resource_name),
-                        class: "btn btn-default pull-right"
-
-  %footer
-    - if display_password_reset_link?
-      = link_to t("devise.passwords.new.forgot_password"), new_password_path(resource_name)
-
-    = link_to t("layouts.application.toggle"), toggle_mobile_path
+#main_stream.stream
+- flash.each do |name, msg|
```

Comment:
Hm, looks like pronto is going crazy here, `name` and `msg` are used the lines below.